src/ExtensionsSample/App.config

```diff
@@ -1,9 +1,5 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-  <connectionStrings>
```

Comment:
Yeah I was backing my own stuff out. I got a little carried away :-)

---
In reply to: [56353289](https://github.com/Azure/azure-webjobs-sdk-extensions/pull/47#discussion_r56353289) [](ancestors = 56353289)